index.html

```diff
@@ -565,6 +565,7 @@
 The time immediately after the user agent receives the last byte of the response or immediately before the transport connection is closed, whichever comes first. The resource here can be received either from <a

```

Comment:
Interesting thanks Philippe! I'll go on the assumption that it was lack-of-keepalive related.